addons/website/static/src/components/navbar/navbar.js

```diff
@@ -29,7 +29,7 @@ patch(NavBar.prototype, 'website_navbar', {
      * @override
      */
     get systrayItems() {
-        if (this.websiteService.currentWebsite && this.websiteService.isRestrictedEditor) {
+        if (this.websiteService.currentWebsite) {
             return websiteSystrayRegistry
```

Comment:
This one comes from QSM when we discussed the spec the other day: need to be sure that this return items and not empty list if all have an unmet visibility condition (even if shouldn't be the case?), in such case we should call the super, otherwise it would replace by Odoo systray by an empty one (basically just hidding the systray)

Reply:
I now fallback to super if the only remaining menues are debug and "burger menu" (the mobile version of the actual menu). Maybe I should include the "Mobile preview" icon in this list, and show it for any user who sees the website-specific menu - not requiring the Restricted Editor...